apps/krystal-bull/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  web:
+    image: bitcoinscala/oracle-server-ui:0.0.1-b6f9101f-SNAPSHOT
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      - ORACLE_SERVER_API_URL=http://10.21.21.61:9998/
+    ports:
+      - 3010:3001
+    networks:
+      default:
+        ipv4_address: 10.21.21.60
+    depends_on: 
+      - oracleserver
+  oracleserver:
+    image: bitcoinscala/bitcoin-s-oracle-server:1.7.0-170-4a526580-SNAPSHOT
+    user: "1000:1000"
+    volumes:
+      - ${APP_DATA_DIR}/data/oracleserver:/home/bitcoin-s/.bitcoin-s
+    environment:
+      - BITCOIN_S_KEYMANAGER_ENTROPY=$APP_SEED  
+    expose:
+      - 9998
```

Comment:
If a port is only accessed from another container, but not exposed to the outside, you don't need this.